catalog/vehicle-single-language-output.schema.json

```diff
@@ -68,12 +64,12 @@
     }}},
     "maker": { "type": "string", "minLength": 1 },
     "model": { "type": "string", "minLength": 1 },
-    "engine": { "type": "string", "minLength": 0 },
-    "finish": { "type": "string", "minLength": 0 },
-    "version": { "type": "string", "minLength": 0 },
-    "interior": { "type": "string", "minLength": 0 },
+    "engine": { "type": ["null","string"], "minLength": 1 },
+    "finish": { "type": ["null","string"], "minLength": 1 },
+    "version": { "type": ["null","string"], "minLength": 1 },
```

Comment:
as seen, the version is not needed
version = engine + finish

Reply:
OK, but in this case, let's remove it from everywhere.

Reply:
done.